frontend/bonds-for-success/src/app/mentee-mentor-table/mentee-mentor-table.component.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { MatTableModule } from '@angular/material/table';
+
+import { MenteeMentorTableComponent } from './mentee-mentor-table.component';
+
+describe('FileInputComponent', () => {
+  let component: MenteeMentorTableComponent;
+  let fixture: ComponentFixture<MenteeMentorTableComponent>;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      declarations: [
```

Comment:
Not sure if this is proper in angular thinking
declarations: [ MenteeMentorTableComponent,],
imports: [MatTableModule,],

Is a lot more readable